src/Poker/Cards.hs

```diff
@@ -198,83 +197,83 @@ instance Bounded Card where
 allCards :: [Card]
 allCards = liftM2 Card allRanks allSuits
 
--- | 'Hand' represents a player's hole cards in a game of Texas Hold\'Em
-data Hand = MkHand !Card !Card
+-- | 'Hole' represents a player's hole cards in a game of Texas Hold\'Em
+data Hole = MkHole !Card !Card
```

Comment:
Consider UnsafeHold as an alternatve constructor name (or, alternatively, stop normalising and just go with Hole)

Reply:
Will follow up separately